app/models/reports/instrument_unavailable_export_raw.rb

```diff
@@ -0,0 +1,90 @@
+require "csv"
+
+module Reports
+
+  class InstrumentUnavailableExportRaw
+
+    attr_reader :facility, :date_start, :date_end
+
+    include CsvExporter
+    include FacilityReservationsHelper
+
+    def initialize(facility:, date_start:, date_end:)
+      @facility = facility
+      @date_start = date_start
+      @date_end = date_end
+    end
+
+    def description
+      "#{facility.name} Instrument Down Export Raw, #{formatted_date_range}"
+    end
+
+    def translation_scope
+      "controllers.instrument_unavailable_reports_controller"
+    end
+
+    private
+
+    def formatted_total_minutes(reservation)
+      duration_in_seconds =
+        ReservationRowDurationCalculator
+        .new(reservation, start_time: date_start.beginning_of_day, end_time: date_end.end_of_day)
+        .duration_in_seconds
+
+      format("%.2f", (duration_in_seconds / 60.0))
+    end
+
+    def default_report_hash
+      {
+        type: -> (reservation) { reservation.type.sub(/Reservation\z/, "") },
```

Comment:
Should we use something more like `model_name.human`? Seems like locales would be a better way of handling this, especially since we're displaying in the type in a couple places.